dist/api_data.py

```diff
@@ -557,6 +557,12 @@ def __ge__(self, other):
     Config('debug_mode', '', r'''
         control the settings of various extended debugging features''',
         type='category', subconfig=[
+        Config('background_compact_expire_time', '86400', r'''
+               how long the compact stats for a file should be tracked for before expiring.''',
+               type='int'),
+        Config('background_compact_skip_time', '60', r'''
+               how long a file should sleep before background compact tries again. ''',
+               type='int'),
```

Comment:
If these are as precise as this, should they be parameters to the WT_SESSION:compact API? @etienneptl as well.

Reply:
I think that will overcomplicate the design of the API. We should aim to tune these as well as we can for them to work in the background without needing to be set.